tests/object_test.py

```diff
@@ -1140,6 +1151,21 @@ def test_datetime_with_tz_copies_refs():
     assert decoded[0] == d0
     assert decoded[1] == d1
 
+def test_with_exclude():
+    """ Does the _jsonpickle_exclude work"""
+    obj = ThingWithExclusion("baz")
+    encoded = jsonpickle.encode(obj)
+    decoded = jsonpickle.decode(encoded)
+    assert decoded.bar == "baz"
+    assert not hasattr(decoded, "foo")
+
+def test_contained_exclusion():
+    """ _jsonpick_exclude should work only on the class it is defined in"""
```

Comment:
Minor typo here, should be _jsonpickle_exclude.

Reply:
Fixed typo